static/templates/connect_result.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
+
+	<title>Connecting...</title>
+
+    <link rel="stylesheet" href="/static/css/application.css" />
+</head>
+
+<body>
+    <div class="wrapper">
+        <div id="main-content" class="inner-wrapper">
+            <div class="row no-border" id="grid">
+            <h3>Connecting...</h3>
+                 <div class="twelve-col box" style="background-color: #eee">
+                    <p>Network will be available if provided credentials are valid. Please check device connection after passed some minutes</p>
```

Comment:
Please change to: 
Device is now attempting to connect to <SSID>. 
You may try to connect to the device through the new <SSID> connection.